src/main/config/logback.xml

```diff
@@ -49,6 +49,35 @@
     </then>
   </if>
 
+  <!-- Enable output metrics logging by creating an environment variable of "ObjectTrace=true" -->
+  <if condition='${ObjectTrace} == true'>
```

Comment:
Might want this change as well
```suggestion
  <if condition="&quot;${ObjectTrace}&quot;.equals(&quot;true&quot;)">
```

https://github.com/NationalSecurityAgency/emissary/pull/549/files